carthage-input-files.xcfilelist

```diff
@@ -0,0 +1,16 @@
+# Carthage input files
```

Comment:
Is `.xcfilelist` a suggested extension for these scripts?

Reply:
Yip - https://developer.apple.com/documentation/xcode_release_notes/xcode_10_release_notes/build_system_release_notes_for_xcode_10

> Run Script build phases now support declaring input and output files in a .xcfilelist file.